docs/DeveloperGuide.md

```diff
@@ -4,25 +4,43 @@
 
 ### Architecture
 
-![Architecture Diagram](diagrams/ArchitectureDiagram.png) 
-
+![Architecture Diagram](diagrams/ArchitectureDiagram.png)
 The ***Architecture Diagram*** given above explains the high-level design of the App. 
 Given below is a quick overview of each component.
 
-**[`FridgeFriend`](https://github.com/AY2021S2-CS2113-T10-1/tp/blob/master/src/main/java/seedu/fridgefriend/FridgeFriend.java)** is driven by a method called `Main`. 
+The Main driver class for the FridgeFriend app is
+named **[`FridgeFriend`](https://github.com/AY2021S2-CS2113-T10-1/tp/blob/master/src/main/java/seedu/fridgefriend/FridgeFriend.java)**. 
 It is responsible for,
 * At app launch: Initializes the components in the correct sequence, and connects them up with each other.
 * At shut down: Shuts down the components and invokes cleanup methods where necessary.
 
-[**`Food`**](#common-classes) represents a collection of classes used by multiple other components.
 
-The rest of the App consists of four components.
 
-* [**`UI`**](#ui-component): The UI of the App.
-* [**`Parser`**](#parser-component): Processes user input, and determines the related command to be executed.
-* [**`Command`**](#command-component): The command executor.
-* [**`Storage`**](#storage-component): Reads data from, and writes data to, the hard disk.
+The rest of the App consists of four components.
 
+* [**`Utilities`**](#utilities-component): The main component containing the key driver classes in 
+  FridgeFriend, such as `Parser`, `UI`, and `Storage`. 
```

Comment:
I think you can just briefly mention that it contains key driver classes. the other details will be in the utilities section

Reply:
Resolved